modular_skyrat/modules/mapping/code/game/turfs/planet_turfs.dm

```diff
@@ -70,9 +70,56 @@
 	vis_contents += snow_overlay
 */
 
+
+///////////////////////////////////////////
+/////////////   OTHER TURFS   /////////////
+///////////////////////////////////////////
+
 /turf/closed/mineral/earth_like
 	icon_state = "rock"
 	turf_type = /turf/open/floor/plating/asteroid
 	baseturfs = /turf/open/floor/plating/asteroid
 	initial_gas_mix = OPENTURF_DEFAULT_ATMOS
 	defer_change = TRUE
+
+/turf/closed/mineral/random/asteroid/rockplanet	//A version that can be used on the mining planet without destroying atmos - starts with Low_Pressure, along with the rest of the planet.
+	name = "iron rock"
+	icon = 'icons/turf/mining.dmi'
+	icon_state = "redrock"
+	smooth_icon = 'icons/turf/walls/red_wall.dmi'
+	base_icon_state = "red_wall"
+	turf_type = /turf/open/floor/plating/asteroid/lowpressure
+	baseturfs = /turf/open/floor/plating/asteroid/lowpressure
+	initial_gas_mix = OPENTURF_LOW_PRESSURE
+	defer_change = TRUE
+
+	mineralSpawnChanceList = list(/obj/item/stack/ore/uranium = 5, /obj/item/stack/ore/diamond = 1, /obj/item/stack/ore/gold = 10,
+		/obj/item/stack/ore/silver = 12, /obj/item/stack/ore/plasma = 20, /obj/item/stack/ore/iron = 40, /obj/item/stack/ore/titanium = 11,
+		/turf/closed/mineral/gibtonite = 3, /obj/item/stack/ore/bluespace_crystal = 1)	//A bit less gibtonite because of higher spawns making it nuts
```

Comment:
```suggestion
	mineralSpawnChanceList = list(
		/obj/item/stack/ore/iron = 40,
		/obj/item/stack/ore/plasma = 20,
		/obj/item/stack/ore/silver = 12,
		/obj/item/stack/ore/titanium = 11,
		/obj/item/stack/ore/gold = 10,
		/obj/item/stack/ore/uranium = 5,
		/turf/closed/mineral/gibtonite = 3,	//A bit less gibtonite because of higher spawns making it nuts
		/obj/item/stack/ore/diamond = 1,
		/obj/item/stack/ore/bluespace_crystal = 1
		)
```
_formatting on web is a fuck_